### ISSUE TITLE
Official VCPKG package

### ISSUE THREAD

**Issue by diehard2** [launchdarkly/cpp-sdks#355]
VCPKG is gaining (has?) a lot of traction. However, the launchdarkly library in there is pretty old. It would be nice to get an official version for this!

**Comment by cwaldren-ld** [comment]
You're correct that the current package is the `c-server-sdk` and not maintained by us - thanks for filing this issue! 

 While I don't have a timeline, we're definitely interested in maintaining a `vcpkg` distribution. 

I'm not sure what it'll take to make this happen but I can guess some CMake improvements will probably be necessary.

_Filed internally as 228167_.

**Comment by diehard2** [comment]
@cwaldren-ld I have a CMakeLists.txt that builds a vcpkg overlay port. Currently only builds the server side sdk, but its in really good shape. let me know if you're interested
